src/ISA/ISA.Json/Ontology.fs

```diff
@@ -17,18 +14,21 @@ module AnnotationValue =
     /// <param name="s"></param>
     /// <param name="json"></param>
     let decoder (options : ConverterOptions) : Decoder<string> =
-        fun s json ->
-            // is there a option to decode force to string?
-            match Decode.int s json with
-            | Ok i -> Ok <| string i
-            | Error _ -> 
-                match Decode.float s json with
-                | Ok f -> Ok <| string f
+        { new Decoder<string> with
+            member this.Decode (s,json) =
+                match Decode.int.Decode(s,json) with
+                | Ok i -> Ok <| string i
                 | Error _ -> 
-                    match Decode.string s json with
-                    | Ok s -> Ok <| s
-                    | Error e -> Error e
+                    match Decode.float.Decode(s,json) with
+                    | Ok f -> Ok <| string f
+                    | Error _ -> 
+                        match Decode.string.Decode(s,json) with
+                        | Ok s -> Ok <| s
+                        | Error e -> Error e       
+        }
 
+            // is there a option to decode force to string?
```

Comment:
this todo is solved?

Reply:
Nope, could open up an issue at Fable, but not sure how you would even call such a function.